lvsfunc/comparison.py

```diff
@@ -73,85 +387,91 @@ def _resample(clip: vs.VideoNode) -> vs.VideoNode:
 
 
 def stack_compare(*clips: vs.VideoNode,
-                  make_diff: bool = False,
-                  height: Optional[int] = None,
-                  warn: bool = True) -> vs.VideoNode:
+                  make_diff: bool = True,
+                  height: int = 288,
+                  warn: Any
+                  ) -> vs.VideoNode:
     """
-    A simple wrapper that allows you to compare two clips by stacking them.
-    You can stack an infinite amount of clips.
+    Best to use when trying to match two sources frame-accurately.
+    Stacks two clips ontop of a MakeDiff clip for lining up clips from multiple sources.
```

Comment:
The first sentence should always describe what a function is doing, not when it could be used.